packages/web/src/common/store/pages/track/sagas.js

```diff
@@ -136,7 +136,6 @@ function* getRestOfLineup(permalink, ownerHandle) {
 function* watchFetchTrack() {
   yield takeEvery(trackPageActions.FETCH_TRACK, function* (action) {
     const { trackId, handle, slug, canBeUnlisted } = action
-    const permalink = `/${handle}/${slug}`
```

Comment:
Do some tracks not have handle or slug? Why the change here

Reply:
oh i just got rid of it because it was no longer needed after the `track.permalink` change below. `Handle` and `slug` are optional params for this action and are not passed in by mobile (since it only gets the `trackId` from the router